slack_bolt/logger/messages.py

```diff
@@ -266,7 +267,37 @@ def warning_unhandled_request(  # type: ignore
         )
     if is_event(req.body):
         # @app.event
-        event_type = req.body.get("event", {}).get("type")
+        event = req.body.get("event", {})
+        event_type = event.get("type")
+        if is_function(req.body):
+            # @app.function
+            callback_id = event.get("function", {}).get("callback_id", "function_id")
+            return _build_unhandled_request_suggestion(
+                default_message,
+                f"""
+@app.function("{callback_id}")
+{'async ' if is_async else ''}def handle_{callback_id}_function(body, event, client, logger):
```

Comment:
It seems that you can have "-" as part of callback_id. In this case, the generated code can have syntax error in Python. I haven't examined what else can be used for callback_id, so replacing all non-alphanumeric characters to "_" (underscore) should be a safe way to deal with such edge cases.

Reply:
Good catch 💯 I've opted to follow a similar pattern found in this file, by hardcoding the function name as `handle_some_function` instead of using the callback_id to dynamically name the function